docs/dev-setup.md

```diff
@@ -305,16 +305,16 @@ The following command will deploy the controllers from CAPI, CABPK and CAPM3 and
 the requested CRDs.
 
 ```sh
-    make deploy
+make deploy
 ```
 
 ### Run the Controller locally
 
 You will first need to scale down the controller deployment :
 
 ```sh
-    kubectl scale -n capm3-system deployment.v1.apps/capm3-controller-manager \
-      --replicas 0
+kubectl scale -n capm3-system deployment.v1.apps/capm3-baremetal-operator-controller-manager --replicas 0
```

Comment:
why do we need to scale down BMO, is not it only CAPM3?

Reply:
Currently the BMO is also deployed, because of the `examples/provider-components/infrastructure-components.yaml` file based on `config/bmo/`